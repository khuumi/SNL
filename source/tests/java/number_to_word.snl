start main:
    num is 15
    do show to "this should print " + (do number_to_word to num) + " to console."
done
