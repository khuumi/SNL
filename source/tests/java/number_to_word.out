this should print 15 to console.
